### PR TITLE
feat: scroll padding

### PR DESCRIPTION
**Pull request by AntonChubarov**
Scroll padding proposal

### CODE REVIEW

**Review by olexandrd — APPROVED** [review]

File: src/css/base.css [hunk 1]
```diff
@@ -21,6 +21,13 @@
 
 html {
   scroll-behavior: smooth;
+  scroll-padding-top: 100px;
+}
+
+@media screen and (min-width: 1280px) {
+  html {
+    scroll-padding-top: 170px;
```
Comment (olexandrd) [inline src/css/base.css:29]: Why `170px`? Lest keep `100px` or some value in range 100-150

Comment (olexandrd) [reply src/css/base.css:29]: 170px;
image

Comment (olexandrd) [reply src/css/base.css:29]: 120px;
image

Comment (AntonChubarov) [reply src/css/base.css]: made 100px for all sizes

**Review by olexandrd — APPROVED** [review]

**Pull request merged**